ci/prow/config.yaml

```diff
@@ -43,14 +43,27 @@ deck:
     gcs_browser_prefix: https://console.cloud.google.com/storage/browser/
     testgrid_config: gs://knative-testgrid/config
     testgrid_root: https://testgrid.knative.dev/
-    viewers:
-      "started.json|finished.json":
-      - "metadata"
-      "build-log.txt":
-      - "buildlog"
-      "artifacts/junit.*\\.xml":
-      - "junit"
     announcement: "The old job viewer, Gubernator, has been deprecated in favour of this page, Spyglass.{{if .ArtifactPath}} For now, the old page is <a href='https://gubernator.knative.dev/build/{{.ArtifactPath}}'>still available</a>.{{end}} Please send feedback to Knative productivity."
+    lenses:
+    - lens:
+        name: metadata
+      required_files:
+      - started.json|finished.json
+    - lens:
+        name: buildlog
+        config:
+          highlight_regexes:
+          - 'panic: test timed out after.*$'
+          - timed out
```

Comment:
can we make the `timed out` a little more specific? Maybe we should ask serving developers whether they want to see this line highlighted or not:
```
istio-system                                                                                               5m36s       Warning   FailedMount                                                            pod/istio-ingressgateway-78c998c64b-c62bd                                                                                            kubelet, gke-kserving-e2e-cls1184-default-pool-9be9e001-2x7d                  MountVolume.SetUp failed for volume "ingressgateway-certs" : couldn't propagate object cache: timed out waiting for the condition                                                                                                                                                                                                                                                                                                                                                            5m36s        1       istio-ingressgateway-78c998c64b-c62bd.15cdfca29dd60f7d   
```

link here(it loads slow): https://prow.knative.dev/view/gcs/knative-prow/logs/ci-knative-serving-istio-1.3-mesh/1184270415527153664#1:build-log.txt%3A21306

Reply:
`timed out` is okay I think, in https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_serving/5760/pull-knative-serving-integration-tests/1184527739000983552, the `timed out` seems to be the real reason for the test failure.